ocs_ci/utility/csr.py

```diff
@@ -169,3 +173,13 @@ def wait_for_all_nodes_csr_and_approve(timeout=900, sleep=10, expected_node_num=
         pending_csrs = get_pending_csr()
         if pending_csrs:
             approve_csrs(pending_csrs)
+        # In vSphere deployment it sometime happes that VM doesn't get ip and
+        # then we need to restart it to make our CI more stable and let the VM
+        # to get IP and continue with loading ignition config. The rester of
+        # the VMs happens only once in reboot_timeout (120 seconds).
+        if (
+            config.ENV_DATA["platform"] == constants.VSPHERE_PLATFORM
+            and start_time >= reboot_timeout
+        ):
+            start_time = time.time()
```

Comment:
Do we need to set `start_time` again? I don't see any more usages.

Reply:
It's in for loop, intention is to run it once a 120 seconds if we will not get proper number of csrs

Reply:
Ah, I didn't extend everything and missed the for loop, makes sense!